cmake/FindVecGeom.cmake

```
@@ -79,4 +79,13 @@ if(VecGeom_FOUND AND VecGeom_CUDA_FOUND AND (TARGET VecGeom::vecgeomcuda OR TARG
   endif()
 endif()
 
+# Save VecGeom_NAV variable for diagnostics
+if(VecGeom_VERSION VERSION_LESS 2.0)
+  if(VECGEOM_USE_NAVINDEX)
+    set(VecGeom_NAV "index")
+  else()
+    set(VecGeom_NAV "path")
+  endif()
+endif()
```

```suggestion
  endif()
else()
  set(VecGeom_NAV "v2 index")
endif()
```
for consistency (not having `VecGeom_NAV` might other means either that this build is using v2 or that this mechanism failed) and it will help prevent confusing if an adventurous user is switching the same Celeritas build area between VecGeom v1 and v2 ...

---

Actually this is deliberately here because I was doing that. VecGeom 2 stores `VecGeom_NAV` as a cache variable, and it's still exported by v1 even though it's not defined anywhere in the CMakeLists, leading to an inconsistent diagnostic downstream in Celeritas. By forcibly providing the navigation type here we can consistently tell whether we're using the index, path, or tuple nav state regardless of the vecgeom version.